app/src/main/java/com/iron/espresso/presentation/home/study/HotKeywordAdapter.kt

```diff
@@ -0,0 +1,22 @@
+package com.iron.espresso.presentation.home.study
+
+import android.view.LayoutInflater
+import android.view.ViewGroup
+import androidx.recyclerview.widget.RecyclerView
+import com.iron.espresso.R
+
+class HotKeywordAdapter(private val items: List<HotKeywordItem>) :
+    RecyclerView.Adapter<ViewHolder>() {
+    override fun onCreateViewHolder(parent: ViewGroup, viewType: Int): ViewHolder {
+        val view = LayoutInflater.from(parent.context)
+            .inflate(R.layout.search_hot_keyword_item, parent, false)
+
+        return ViewHolder(view)
+    }
+
+    override fun getItemCount(): Int = items.size
```

Comment:
```suggestion
    override fun getItemCount(): Int = 
        items.size
```
https://kotlinlang.org/docs/reference/coding-conventions.html#expression-body-formatting